python/tvm/relay/expr_functor.py

```diff
@@ -36,9 +36,8 @@ def __init__(self):
     # pylint: disable=no-else-return
     def visit(self, expr):
         """Apply the visitor to an expression."""
-        found = self.memo_map.get(expr)
-        if found:
-            return found
+        if expr in self.memo_map:
```

Comment:
Try this
```python3
found = self.memo_map.get(expr, None)
if found is not None:
```

Reply:
this exactly do not work.

Reply:
the problem is get might return None

Reply:
(because ExprVisitor store None in them)

Reply:
I see. Thanks!